capstone/src/main/webapp/youtube_analyzer.html

```diff
@@ -70,7 +70,8 @@ <h1>Youtube Analysis</h1>
         <button onclick="callYoutube()" class="btn btn-primary">Submit</button>
         <button onclick="getKeywordSearchResults()" class="btn btn-primary" id="keywordSearch">Keyword Search</button>
       </form><br>
-      <div id="comment-list"></div>
+       <div id="loading-container"></div>
+       <div id="comment-list"></div>
        <div id="chart-container"></div>
        <div id="table-container"></div>
```

Comment:
nit: Do these lines have an extra indent? Also, is it possible to remove the `<br>` tag above with some CSS style changes?

Reply:
addressed in "fix indenting and spacing"